org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_detail.scss

```diff
@@ -65,11 +65,28 @@
 
   .crm-summary-block-wrap {
     width: 100%;
+    margin-top: 0;
   }
 
   #crm-contactname-content {
     border: none !important;
+    height: 60px;
     max-height: 60px;
+
+    .crm-summary-display_name {
+      padding-top: 22px;
+    }
+  }
+
+  .crm-summary-contactname-block {
+    display: flex;
```

Comment:
Flexbox is not fully supported by IE11 (http://caniuse.com/#feat=flexbox), so either you check that this solution works fine in it, or you change the solution so that it doesn't rely on flexbox